accepted/PSR-12-extended-coding-style-guide.md

```diff
@@ -232,7 +232,7 @@ declare(ticks=1) {
 
 ## 4. Classes, Properties, and Methods
 
-The term "class" refers to all classes, interfaces, and traits.
+The term "class" refers to all classes, anonymous classes, interfaces, and traits.
```

Comment:
I don't think this is right, because the position of the opening brace for genuine classes is different to that of anonymous classes.

Reply:
PR is legit, but I would also remove this part which only make thing more confusing by recursive definition.